src/query/string_opfunc.c

```diff
@@ -3674,6 +3674,73 @@ db_json_array_insert (DB_VALUE * result, DB_VALUE * arg[], int const num_args)
   return NO_ERROR;
 }
 
+int
+db_json_contains_path (DB_VALUE * result, DB_VALUE * arg[], const int num_args)
+{
+  bool find_all = false;
+  bool exists = false;
+  int error_code = NO_ERROR;
+  JSON_DOC *doc = NULL;
+  db_make_null (result);
+
+  if (DB_IS_NULL (arg[0]) || DB_IS_NULL (arg[1]))
+    {
+      return NO_ERROR;
+    }
+
+  doc = db_get_json_document (arg[0]);
+  const char *find_all_str = db_get_string (arg[1]);
+
+  if (strcmp (find_all_str, "all") == 0)
```

Comment:
I'd like to ask you to confirm case issue of `all` and `one`.
Others look good.

Reply:
It mimics mySql behavior of raising an error if this argument is not _all_/_one_

Reply:
Then, let's accept it.